### ISSUE TITLE
Questions about "Masked Scene Contrast" and outdoor segmentation

### ISSUE THREAD

**Issue by meidachen** [Pointcept/Pointcept#19]
Hi, thanks for sharing these amazing works! The codebase is very well organized, and the publications are so interesting. 

While reading your work on Masked Scene Contrast, I wondered what was the reason that this work focused on using SpConv instead of using transforms your team has previously designed? Have you tested any pre-training on transformers? From 1d and 2d problems, it seems like transformers may benefit more from pretraining. It would be great if you could provide more insights. 

Another question is related to semantic segmentation of outdoor scenes, I tried to use transformers for outdoor scene segmentation on our data set (https://www.stpls3d.com), and I have tried many different settings, but still didn't get good results (a few points lower than KpConv, RandLA, etc.). I also saw most of the transformers were tested on indoor datasets but not outdoors in publications, so I couldn't think of a good reason why this is the case (maybe something is wrong with our dataset? maybe the feature extracted in transformers is not local enough for outdoor scenes?). I would really appreciate it if you could share some of your thoughts about outdoor scene segmentation using transformers.

Thank you for your help in advance!

**Comment by Gofinge** [comment]
```
The reason that this work focused on using SpConv 
```
Because SpConv is fast and memory efficient, we will also release a pre-training config for PTv2. I only had limited computing resources then, and I did not tune much for MSC + PTv2, although the maximum performance is impressive but still unstable. 

```
 It seems like transformers may benefit more from pretraining.
```
Yes, you are right, especially when we need to scale up the model. My next work focuses on multi-dataset joint training to further enable large-scale pretraining for 3D scenes, and I will further explore the combination of PTv2 and pre-training framework.

```
semantic segmentation of outdoor scenes
```
We recently found that PTv2 can also achieve SOTA performance without changing model architecture (align augmentation with SOTA model in outdoor) in outdoor semantic tasks (SemanticKITTI, nuScene), we will update the codebase and config.
